src/__tests__/lib/mailtrap-client.test.ts

```diff
@@ -664,20 +706,6 @@ describe("lib/mailtrap-client: ", () => {
   });
 
   describe("get testing(): ", () => {
-    it("rejects with Mailtrap error, when `testInboxId` is missing.", () => {
```

Comment:
It would also be nice to have a test that the absence of `testInboxId` doesn't raise an error when `client.testing` is called.

Reply:
And remove `testInboxId` from all client setup cases where it's not needed anymore (like [here](https://github.com/railsware/mailtrap-nodejs/pull/70/files#diff-5e393473a230a554c92756272d9f99b28371cea5125bdedc5c980d66bf22e3bfR728)).